tractor/_actor.py

```diff
@@ -575,6 +576,17 @@ async def _async_main(
                         await chan.connect()
                         # initial handshake, report who we are, who they are
                         await self._do_handshake(chan)
+
+                        if self._spawn_method == "trio":
+                            # recieve additional init params
+                            parent_data = await chan.recv()
+                            for attr, value in parent_data.items():
```

Comment:
Yeah, seeing this now I'm wondering if it's a bit too *monkey-patchy*.
I think it's fine for now but i'd be interesting to see if we can make this a little more stringent (strictly checked against pre-defined class attrs) using a package like `attrs`.

We can maybe make this a todo for later since I'm sure there's other classes that could be simplified as well. I feel like `attrs` is an acceptable dep since `trio` already uses it as well.

Reply:
Deferred to #139 .